### PR TITLE
feat(editors): Add new Che Editor plugin: Intellij Community

### PR DESCRIPTION
**Pull request by benoitf**
Add Che Editor for intellij community editor

### What does this PR do?

Fixes https://github.com/eclipse/che/issues/17979

Change-Id: Ie6b9fa4cef57532e84f95c88bfca1bd5f4e8e70a
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

File: v3/plugins/che-incubator/intellij-community/2020.2.2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+title:  IntelliJ IDEA Community Edition (in browser using noVNC) as editor for Eclipse Che
+description:  IntelliJ IDEA Community Edition running on the Web with noVNC
+icon: https://resources.jetbrains.com/storage/products/intellij-idea/img/meta/intellij-idea_logo_300x300.png
+category: Editor
+repository: https://github.com/che-incubator/che-editor-intellij-community
+firstPublicationDate: "2020-09-29"
+spec:
+  endpoints:
+  # hack (use dirigible endpoint name as it is part of liveness probes)
+   -  name: "dirigible"
```
Comment (azatsarynnyy) [inline v3/plugins/che-incubator/intellij-community/2020.2.2/meta.yaml:16]: Is it mandatory to name it as `dirigible`?

Comment (benoitf) [reply v3/plugins/che-incubator/intellij-community/2020.2.2/meta.yaml:16]: @azatsarynnyy it's a workaround before che server can accept intellij liveness probe

without that (or using another probe name), you can be redirected to the editor before it's really listening, then you've a openshift/k8s missing routes error and you need to refresh the full page

Comment (azatsarynnyy) [reply v3/plugins/che-incubator/intellij-community/2020.2.2/meta.yaml:16]: There's a [hardcoded list](https://github.com/eclipse/che/blob/982a0a148cf6d635da41726c984abcba12dd9d1a/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L166) on ws-master side.
Now I see how Che distinguishes such endpoints from the other ones :slightly_smiling_face: 
Thanks for the explanation!

**Review by ericwill — APPROVED** [review]

**Review by azatsarynnyy — APPROVED** [review]
Thank you!

**Pull request merged**